openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -109,4 +109,11 @@ jQuery(function () {
         import(/* webpackChunkName: "editions-table" */ './editions-table')
             .then(module => module.initEditionsTable());
     }
+    // Enable any carousels in the page
+    if (carouselElements.length) {
+        import(
```

Comment:
Place these on a single line (like above) for consistency. The visual similarity makes it easier to read (you don't have to ready each block; because they have the same "shape", you only have to read one and will be able to assume what the others do).

Reply:
Done.